### DISCUSSION TITLE
https://greendolphindance.com/2024/01/22/%E5%88%B6%E4%BD%9Chexo%E7%83%AD%E5%8A%9B%E5%9B%BE/

### DISCUSSION THREAD

**Discussion by giscus** [greendolphindance/greendolphindance.github.io#22] [General]
# https://greendolphindance.com/2024/01/22/%E5%88%B6%E4%BD%9Chexo%E7%83%AD%E5%8A%9B%E5%9B%BE/

Back to the rough ground!

https://greendolphindance.com/2024/01/22/%E5%88%B6%E4%BD%9Chexo%E7%83%AD%E5%8A%9B%E5%9B%BE/

**Comment by rebron1900** [comment]
请问如果只想暂时最近一周的该如何做？

**Reply by rebron1900** [reply]
展示

**Reply by greendolphindance** [reply]
问一下GPT吧！我也看不懂这些代码TT

**Comment by erispyu** [comment]
安利一下我自制的hexo插件，一行代码即可在hexo任意md文件插入站点统计图表（目前实现了heatmap热力图、饼图、雷达图，后续会持续更新）。

Demo: https://blog.erispyu.fun/

GitHub: https://github.com/erispyu/hexo-stats-echarts/blob/main/docs/zh-CN/README.md

欢迎试用、评论，觉得不错的话，也可以给一个star🌟，感谢
